ui/src/components/TextInput.js

```diff
@@ -11,8 +11,7 @@ const TextInputWrapper = styled('div')`
 `;
 
 const RefArrangerTextInput = React.forwardRef((props, ref) => (
-  <div ref={ref}></div>
-  // <ArrangerTextInput componentRef={ref} shouldAutoFocus={false} {...props} />
+  <ArrangerInput ref={ref} shouldautofocus={'false'} {...props} />
```

Comment:
`shouldautofocus` is not a props of Input (which in this case is a wrapper for plain old `<input>`), and since you want it "off" (i.e. false), can well remove it.
```suggestion
  <ArrangerInput {...props} ref={ref}  />
```
Also, I'm placing ref after the spread, so nothing being passed down in there will accidentally overwrite the one you're creating here.

Reply:
Noticed there's an internal TextInput used elsewhere, while this one is imported from Arranger. any idea what the difference is, and whether HCMI could stick to using its internal component for this here?

Arranger's Input was not meant to be used outside, so I'm curious why they seem to "need" it.

Reply:
Very good question
This current file `/components/TextInput` defines a TextInput which is used in the `SidebarTextSearch` component which is used in the top 3 home page search bars (Model/Gene/Variant)
I don't know that there is a real need to use an Arranger Input, I replaced the previous Arranger 2 implementation here without much investigation
But, I couldn't find a separate `internal TextInput` definition either so this fills a gap unless you have another recommendation

There is a ModelName Search component using Arranger QuickSearch but I just noticed the file has a deprecation comment (from 5 years ago) and is unused in the repo, I'll delete it:
https://github.com/nci-hcmi-catalog/portal/pull/1099/files#diff-507dc859f8ca4813059349d8cb5f9ae45a6a1c8bb7cf9c5b602eaf4bb6eb1ce5